File: angular/src/app/doctor/models/lockups-dto.ts


import { EntityDto } from '@abp/ng.core';

export class LockupsDto extends EntityDto {
  country: any[];
  state: any[];
  city: any[];
  doctorSpecialty: any[];
  doctorTitle: any[];

  constructor(initialValues: Partial<LockupsDto> = {}) {
    super(initialValues);
  }
}

File: aspnet-core/src/ERP.Healthcare.Application.Contracts/Doctors/LockupsDto.cs
using System.Collections.Generic;
using Volo.Abp.Application.Dtos;

namespace ERP.Healthcare.Doctors
{
    public class LockupsDto : EntityDto
    {
        public List<CountryDto> Country { get; set; }
        public List<StateDto> State { get; set; }
        public List<CityDto> City { get; set; }
        public List<DoctorSpecialtyDto> DoctorSpecialty { get; set; }
        public List<DoctorTitleDto> DoctorTitle { get; set; }
    } 
}
